modules/metastorage-server/src/main/java/org/apache/ignite/internal/metastorage/server/raft/MetaStorageListener.java

```diff
@@ -418,7 +418,7 @@ public boolean onSnapshotLoad(Path path) {
     public void onShutdown() {
         try {
             storage.close();
-        } catch (Exception e) {
+        } catch (RuntimeException e) {
```

Comment:
Well, that's actually very confusing. Almost no one will handle RuntimeException.